kleinanzeigen.py

```diff
@@ -296,7 +334,13 @@ def post_ad(driver, ad, interactive):
     text_area = driver.find_element_by_id('pstad-price')
     if ad["price_type"] != 'GIVE_AWAY':
         text_area.send_keys(ad["price"])
-    price = driver.find_element_by_xpath("//input[@name='priceType' and @value='%s']" % ad["price_type"])
+        try:
+            price = driver.find_element_by_xpath("//input[@name='priceType' and @value='%s']" % ad["price_type"])
+        except NoSuchElementException:
+            try:
+                price = driver.find_element_by_xpath("//select[@name='priceType']/option[@value='%s']" % ad["price_type"])
+            except NoSuchElementException:
+                raise Exception('Cannot find price type selection!')        
     price.click()
```

Comment:
Either the indention of the try/catch block is wrong or the indention of price.click(). 

Your change will result in a null pointer exception in case price_type != GIVE_AWAY

Reply:
@donwayo By merging this PR you now introduced the mentioned error.